src/App.jsx

```diff
@@ -1,64 +1,79 @@
 import './App.css'
 import SectionContentWrapper from './components/sectionContentWrapper/SectionContentWrapper.jsx';
-import novilogo from './assets/novi-logo-color-transparent.png';
 import Navbar from './components/navbar/Navbar.jsx';
+import Button from './components/button/Button.jsx';
 
 function App() {
 
-	return (
-		<>
-			<header className="header">
-				<div className="header-content">
-					<Navbar/>
-					<h1>Dé hogeschool voor ICT</h1>
-				</div>
-			</header>
-			<main>
-				<section className="courses-section">
-					<SectionContentWrapper>
-						<h2>Onze hbo ICT-opleidingen</h2>
-						<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aut illum impedit inventore iste
-							nostrum omnis quae, quam quis rem ut.</p>
-					</SectionContentWrapper>
+    return (
+        <>
+            <header className="header">
+                <div className="header-content">
+                    <Navbar/>
+                    <h1>Dé hogeschool voor ICT</h1>
+                </div>
+            </header>
+            <main>
+                <section className="courses-section">
+                    <SectionContentWrapper>
+                        <h2>Onze hbo ICT-opleidingen</h2>
+                        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aut illum impedit inventore iste
+                            nostrum omnis quae, quam quis rem ut.</p>
+                        <Button name="button1"
```

Comment:
Als je button veel props heeft is het netter om de enters goed te gebruiken:
<img width="781" alt="Screenshot 2024-12-05 at 10 18 24" src="https://github.com/user-attachments/assets/0e90fe53-0641-4b67-994b-f5d9af328bfc">